### DISCUSSION TITLE
Integrate with Flask

### DISCUSSION THREAD

**Discussion by Gl0deanR** [filebrowser/filebrowser#1527] [Q&A]
Hi, I have a live Flask app running on a ubuntu server on nginx and I was wondering if it is possible to run filebrowser as well on the same server. I managed to install it but I can't figure out what settings I must to do to make it run. 
Thank you!

**Comment by o1egl** [comment]
Follow docs on those pages https://filebrowser.org/installation https://filebrowser.org/cli/filebrowser

**Reply by Gl0deanR** [reply]
Thanks, will do and get back after.
